core/src/main/java/org/geysermc/geyser/configuration/GeyserConfiguration.java

```diff
@@ -96,6 +96,9 @@ public interface GeyserConfiguration {
 
     boolean isForceResourcePacks();
 
+    // TODO configurate - till then, api only
+    //List<String> getResourcePackUrls();
```

Comment:
I know what these are doing here, but plzzzz remove. It'll mean one less merge conflict.

Reply:
Sure, I'll remove them :)